src/presets.asm

```diff
@@ -753,7 +753,7 @@ preset_scroll_fixes:
     LDA !sram_custom_preset_slot
     ASL : XBA
     CLC : ADC #$31E9 : TAX       ; X = Source
-    LDY #$CD52 : LDA #$0031      ; Y = Destination, A = Size-1
+    LDY #$CD51 : LDA #$0031      ; Y = Destination, A = Size-1
```

Comment:
An easy mistake to overlook, but I'm surprised we hadn't noticed this until now.  It was apparent something was wrong when I saved and loaded a custom preset at Pancakes and Wavers.  Just outside Wrecked Ship at East Ocean was also not good.

Fortunately the mistake was on load and the scroll data that was saved is correct.

In other news scrolling sky room backgrounds look good when loading presets now.  I tried turning scrolling sky back on but same issue as before, so leaving scrolling sky off.  It looks fine as is.